app/views/types/new.html.erb

```diff
@@ -39,8 +39,10 @@ See COPYRIGHT and LICENSE files for more details.
   end
 %>
 
-<%= error_messages_for 'type' %>
-
-<%= form_for controller.type, builder: TabularFormBuilder, lang: current_language  do |f| %>
-  <%= render partial: 'types/form/settings', locals: { f: f } %>
-<% end %>
+<%=
+  render WorkPackages::Types::SettingsComponent.new(
+    @type,
+    copy_workflow_from: params.dig(:type, :copy_workflow_from),
+    params:
```

Comment:
:red_circle: Passing `params` into the component is a debugging left-over, isn't it?

Reply:
yes, definitely, good catch!